scripts/release.js

```diff
@@ -61,12 +62,18 @@ function versionTagAndPublish() {
   const currentPublished = findCurrentPublishedVersion();
   console.log(`current published version: ${currentPublished}`);
 
-  const version = semver.gt(packageVersion, currentPublished) ? packageVersion : semver.inc(currentPublished, VERSION_INC);
+  let version;
+  if (isSnapshotBuild) {
+    version = `${currentPublished}-snapshot.${process.env.BUILD_ID}`;
```

Comment:
'findCurrentPublishedVersion()' will give you the latest of the 'release' branch (published), but you want the latest of 'master' branch (merged), no?

Reply:
No, that's fine. 
I want the latest published release version. 
This way I know a specific snapshot comes after a specific released version